### PR TITLE
Fix issue #107

### PR DESCRIPTION
**Pull request by SeanRBurton**
Sorry about this.

### CODE REVIEW

**Comment by BurntSushi** [comment]
@SeanRBurton Could you please explain the problem/fix? Thanks.

File: src/shrink.rs [hunk 1]
```diff
@@ -303,10 +303,12 @@ impl Shrinker for StdShrinker {
             &mut StdShrinkerBody::Div8(ref mut shrinker) =>
```
Comment (bluss) [inline src/shrink.rs]: I guess that's a fix for another time, but this would be in more usual style if all the `&mut` were dropped.

**Comment by bluss** [comment]
you're on the ball, that was quick!

**Comment by BurntSushi** [comment]
@SeanRBurton I'm not sure if you saw my previous comment, but could you please explain the fix in the commit message?

**Comment by bluss** [comment]
@SeanRBurton Hi, it would be great if you could clarify this. I also think automatic shrinking sounds awesome (I've got some graphs that are way too complicated to write the shrinker for).

**Comment by BurntSushi** [comment]
@SeanRBurton Unfortunately, I don't yet understand the auto shrinking code which means I can't maintain it. I unfortunately had to revert #100 as a consequence. I'd be happy to bring the changes back in, but I'll need your help understanding them first.

**Pull request closed without merging**